lxd/storage/drivers/driver_dir.go

```diff
@@ -249,16 +249,11 @@ func (d *dir) CreateVolume(vol Volume, filler func(mountPath, rootBlockPath stri
 			return err
 		}
 
-		if shared.PathExists(rootBlockPath) {
-			_, err = shared.RunCommand("qemu-img", "resize", rootBlockPath, fmt.Sprintf("%d", blockSizeBytes))
```

Comment:
@monstermunchkin if we do this need this, then we should also add the `-f raw` argument here as qemu-img says its safer.